shift_detector/precalculations/count_vectorizer.py

```diff
@@ -0,0 +1,87 @@
+import pandas as pd
+from shift_detector.precalculations.precalculation import Precalculation
+from nltk.corpus import stopwords as nltk_stopwords
+from sklearn.feature_extraction.text import CountVectorizer as CountVectorizer_sklearn
+from shift_detector.utils.column_management import ColumnType
+
+
+class CountVectorizer(Precalculation):
+
+    def __init__(self, cols, stop_words='english', max_features=None):
```

Comment:
I will not write everything again, please have a look at the review comments of `LdaGensimTokenizer`. 🙏